setup.py

```diff
@@ -46,7 +46,7 @@ def read_requirements(name):
         'Topic :: Software Development :: Libraries',
         'Topic :: Software Development :: Libraries :: Python Modules'
     ],
-    packages=setuptools.find_packages(),
+    packages=['sprockets.mixins.amqp'],
```

Comment:
Why not just do `setuptools.find_packages(exclude=['tests'])`? Reasoning is, I'm not 100% if leaving out `['sprockets', 'sprockets.mixins']` will change the behavior in some way?